tutorials/tutorial_lesson_04_debugging_2.html

```diff
@@ -343,75 +373,6 @@
         <span class="hl slc">// Realize the function over an 8x8 region.</span>
         <span class="hl kwd">printf</span><span class="hl opt">(</span><span class="hl str">&quot;Evaluating gradient</span><span class="hl esc">\n</span><span class="hl str">&quot;</span><span class="hl opt">);</span>
         Buffer<span class="hl opt">&lt;</span><span class="hl kwb">int</span><span class="hl opt">&gt;</span> output <span class="hl opt">=</span> gradient<span class="hl opt">.</span><span class="hl kwd">realize</span><span class="hl opt">(</span><span class="hl num">8</span><span class="hl opt">,</span> <span class="hl num">8</span><span class="hl opt">);</span>
-        <a onclick='return toggle("_39");'><b id=_39-show>// Click to show output ...</b></a><div id=_39 style= "display:none; background:#333333; color:#ffffff"><b>
- > Begin pipeline gradient.0()
- > Store gradient.0(0, 0) = 0
- > Store gradient.0(1, 0) = 1
- > Store gradient.0(2, 0) = 2
- > Store gradient.0(3, 0) = 3
- > Store gradient.0(4, 0) = 4
- > Store gradient.0(5, 0) = 5
- > Store gradient.0(6, 0) = 6
- > Store gradient.0(7, 0) = 7
- > Store gradient.0(0, 1) = 1
- > Store gradient.0(1, 1) = 2
- > Store gradient.0(2, 1) = 3
- > Store gradient.0(3, 1) = 4
- > Store gradient.0(4, 1) = 5
- > Store gradient.0(5, 1) = 6
- > Store gradient.0(6, 1) = 7
- > Store gradient.0(7, 1) = 8
- > Store gradient.0(0, 2) = 2
- > Store gradient.0(1, 2) = 3
- > Store gradient.0(2, 2) = 4
- > Store gradient.0(3, 2) = 5
- > Store gradient.0(4, 2) = 6
- > Store gradient.0(5, 2) = 7
- > Store gradient.0(6, 2) = 8
- > Store gradient.0(7, 2) = 9
- > Store gradient.0(0, 3) = 3
- > Store gradient.0(1, 3) = 4
- > Store gradient.0(2, 3) = 5
- > Store gradient.0(3, 3) = 6
- > Store gradient.0(4, 3) = 7
- > Store gradient.0(5, 3) = 8
- > Store gradient.0(6, 3) = 9
- > Store gradient.0(7, 3) = 10
- > Store gradient.0(0, 4) = 4
- > Store gradient.0(1, 4) = 5
- > Store gradient.0(2, 4) = 6
- > Store gradient.0(3, 4) = 7
- > Store gradient.0(4, 4) = 8
- > Store gradient.0(5, 4) = 9
- > Store gradient.0(6, 4) = 10
- > Store gradient.0(7, 4) = 11
- > Store gradient.0(0, 5) = 5
- > Store gradient.0(1, 5) = 6
- > Store gradient.0(2, 5) = 7
- > Store gradient.0(3, 5) = 8
- > Store gradient.0(4, 5) = 9
- > Store gradient.0(5, 5) = 10
- > Store gradient.0(6, 5) = 11
- > Store gradient.0(7, 5) = 12
- > Store gradient.0(0, 6) = 6
- > Store gradient.0(1, 6) = 7
- > Store gradient.0(2, 6) = 8
- > Store gradient.0(3, 6) = 9
- > Store gradient.0(4, 6) = 10
- > Store gradient.0(5, 6) = 11
- > Store gradient.0(6, 6) = 12
- > Store gradient.0(7, 6) = 13
- > Store gradient.0(0, 7) = 7
- > Store gradient.0(1, 7) = 8
- > Store gradient.0(2, 7) = 9
- > Store gradient.0(3, 7) = 10
- > Store gradient.0(4, 7) = 11
- > Store gradient.0(5, 7) = 12
- > Store gradient.0(6, 7) = 13
- > Store gradient.0(7, 7) = 14
- > End pipeline gradient.0()
-
-</b></div>
```

Comment:
Looks like these traces all disappeared, which means the snippets weren't successfully included.

Reply:
Where/how are the snippets produced? In Gitter you mentioned "To generate the snippets you run generate_snippets in tutorials/figures" but I don't see any scripts of any sort there -- gen_tutorials.sh appears to be the only script in the whole repo.

Reply:
tutorials/figures in the main Halide repo. The scripts are weirdly split between the two repos.